src/config_parser/index.ts

```diff
@@ -1,52 +1,57 @@
-import { Context } from 'probot' // eslint-disable-line no-unused-vars
-import Webhooks from 'probot/node_modules/@octokit/webhooks' // eslint-disable-line no-unused-vars
-import { OwnershipRules, DirectoryMatchingRule } from '../types' // eslint-disable-line no-unused-vars
+import { Context } from "probot"; // eslint-disable-line no-unused-vars
+import Webhooks from "probot/node_modules/@octokit/webhooks"; // eslint-disable-line no-unused-vars
+import { OwnershipRules, DirectoryMatchingRule } from "../types"; // eslint-disable-line no-unused-vars
 
-export function parseOwnershipRules (rules: any, context: Context<Webhooks.WebhookPayloadPullRequest>): OwnershipRules {
+export const parseOwnershipRules = (
+  rules: any,
```

Comment:
Codacy found an issue: [Type declaration of 'any' loses type-safety. Consider replacing it with a more precise type.](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6083797)